### ISSUE TITLE
why DQN use kernel size 8 ?

### ISSUE THREAD

**Issue by opentld** [dennybritz/reinforcement-learning#222]
The codes in DQN
conv1 = tf.contrib.layers.conv2d(X, 32, 8, 4, activation_fn=tf.nn.relu)
the kernel size is 8.  Why even numbers are used as convolution kernels ?
and the codes:
state = env.reset()
state = state_processor.process(sess, state)
state = np.stack([state] * 4, axis=2)
Why is the number of input channels 4 ？ Why input the same 4 images instead of 1?

Hope you answer... @dennybritz
